assets/scss/base.scss

```diff
@@ -361,73 +299,12 @@ a.v-btn {
 }
 // Common button end
 
-.h-divider {
-  margin: 0;
-  border-color: #e2e8ee;
+.theme--light.v-divider.h-divider {
+  border-width: 0 0 3px 0;
   border-style: solid;
-  border-width: 3px;
-  box-shadow: 0 1px 1px $gray4, 0 -1px 1px $gray4;
+  border-color: $gray2-5;
 }
 
-
-@media only screen and (max-width: 640px) {
```

Comment:
These didn't need to be global so I moved them to the index page where they're needed.